subsys/bluetooth/host/Kconfig

```diff
@@ -183,6 +184,13 @@ config BT_CONN_TX_MAX
 	  Maximum number of pending TX buffers that have not yet
 	  been acknowledged by the controller.
 
+config BT_CONN_DEFER_TX_CB
+	bool "Defer transmission complete callback to system workqueue"
```

Comment:
Making this configurable doesn't feel like the right solution (or at least sufficient). It has been an oversight *not* to have the TX thread size app-configurable, yet still have a callback from the thread going to the app. So either this option should eliminate the callback support all together, or then the TX thread size should be made app-configurable. Simplest solution would be to just always do the deferring and remove this new option.

Reply:
Sure, I was debating about this, just thought Id use a lot more elements on bt_conn_tx but it end up just be 8 bytes (56 bytes in total with current default of BT_CONN_TX_MAX).